Assets/GothicVR/Scripts/Creator/VobCreator.cs

```diff
@@ -203,19 +207,43 @@ private void CreateZoneMusic(PxVobZoneMusicData vob)
             soundCreator.Create(vob, parentGos[vob.type]);
         }
 
+        private void CreateTriggerChangeLevel(PxVobTriggerChangeLevelData vob)
```

Comment:
Just an idea: Isn't it similar to creating a Spot? Could you potentially call CreateSpot() and change a few settings from the returned GO?

Reply:
When we will implement the rest of the triggers we will have a separate class for creating triggers, and this will fall under that